Makefile

```diff
@@ -0,0 +1,62 @@
+PARENT_BRANCH_SUFFIX ?= $(strip $(shell git branch --show-current | cut -d'-' -f2))
```

Comment:
It'd be great to have comments and a help section for the commands that can be used with this makefile.

Reply:
I am working on adding a README file. I can make these changes in the follow up PR, so that it would be easy to propagate changes to all branches using cherrypick.